Rakefile

```diff
@@ -87,7 +87,17 @@ namespace :dependencies do
     task :apply do
       next unless Dir.exist?(File.join(Dir.home, '.mobile-secrets/.git')) || ENV.key?('CONFIGURE_ENCRYPTION_KEY')
 
-      sh('FASTLANE_SKIP_UPDATE_CHECK=1 FASTLANE_ENV_PRINTER=1 bundle exec fastlane run configure_apply force:true')
+      command_compontents = %w[
+        FASTLANE_SKIP_UPDATE_CHECK=1
+        FASTLANE_HIDE_CHANGELOG=1
+        FASTLANE_HIDE_PLUGINS_TABLE=1
+        FASTLANE_ENV_PRINTER=1
+        FASTLANE_SKIP_ACTION_SUMMARY=1
+        FASTLANE_HIDE_TIMESTAMP=1
+        bundle exec fastlane run configure_apply force:true
+      ]
+
+      sh(command_compontents.join(' '))
```

Comment:
Replace `.join(' ')` with [`.shelljoin`](https://ruby-doc.org/stdlib-2.5.1/libdoc/shellwords/rdoc/Shellwords.html#method-c-shelljoin), for safety? You may need to `require 'shellwords'` first.

Reply:
TIL. Neat.

Reply:
I tried this would and it has the unexpected result of escaping the `=` in the env var declarations:

<img width="1592" alt="Screen Shot 2022-07-22 at 2 48 07 pm" src="https://user-images.githubusercontent.com/1218433/180364921-29fbadd1-8794-43f2-b968-8e6dcd6e1e36.png">

Do you know if there is a way of avoiding this? I couldn't see options to pass to the method in the docs.